test/ui/simple-order.test.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const { By, until } = require('selenium-webdriver');
+
+const driverUtil = require('../../utils/driver');
+const { uiBaseUrl } = require('../../utils/base-urls');
+
+const customerService = require('../../services/customers');
+const emailService = require('../../services/email');
+const orderService = require('../../services/orders');
+const productService = require('../../services/products');
+const selectUtil = require('../../utils/select-utils');
+const timingUtils = require('../../utils/timing-utils');
+
+describe('Test create a return for 1 item', () => {
+  let orderId;
+  let orderName;
+  let customerId;
+  let customerEmail;
+  let testUrl;
+  let driver;
+
+  beforeAll(async () => {
+    driver = await driverUtil.buildChromeDriver({ implicit: 5000 });
+    const product = await productService.getProduct('2000');
+    customerEmail = await emailService.getEmail();
+    const config = {
+      email: customerEmail,
+      customer: {
+        first_name: 'Smoke Test - Simple Order',
+        last_name: 'Smoke Test',
+        email: customerEmail
+      },
+      lines: [
+        {
+          variant_id: product.variant_id,
+          product_id: product.id,
+          quantity: 1,
+          fulfillable_quantity: 1,
+          fulfillment_status: 'fulfilled',
+          title: product.title,
+          variant_title: product.variant_title,
+          price: product.price,
+          sku: product.sku,
+          barcode: product.barcode,
+          vendor: '1999',
```

Comment:
Using a vendor that doesn't exist in Mirakl keeps them from potentially showing up in any reports that may incidentally run in non-prod AND keeps us incidents from being opened